### ISSUE TITLE
HYD-DC1 not licensed

### ISSUE THREAD

**Issue by kheven** [MicrosoftDocs/microsoft-365-docs#379]
Hi guys,
Downloaded the 1903 lab kit and was frustrated to see the Domain Controller - HYD-DC1 - shutdown for no reason a few times.
Looking through the event logs, I saw this error message:

Event 1074
Reason Code: 0x80000000
 Shutdown Type: shutdown
 Comment: The license period for this installation of Windows has expired. The operating system is shutting down.

Is there something wrong the the DC VM that's been included in the 1903 lab?

**Comment by mypil** [comment]
@officedocsbot assign @mypil

**Comment by mypil** [comment]
@kheven - Thank you for submitting feedback.

I think the best way forward is if you can open a service ticket in your tenant so this can get resolved ASAP. Based on outcome let me know if it is something that can be called out in the docs.

Please follow this link to contact support for business products:  **https://docs.microsoft.com/office365/admin/contact-support-for-business-products**

If you don't mind please keep us posted here on the resolution and I really want to get this information you are discovering back into the docs.

**Comment by kheven** [comment]
@mypil Thanks for getting back to me.
I wasn't able to log a call yesterday while at work.
When I got home to check the Lab environment, DC1 seems to be properly registered as an evaluation with 150 or so days remaining.
Perhaps it needed internet gateway to be working to validate the license? Initially my External vSwitch wasn't configured with the correct external NIC. Once I corrected that, and provided internet access to DC1, it seems to have resolved itself.
